### PR TITLE
The path to 1.0.0: Add a PR template

### PR DESCRIPTION
**Pull request by juztamau5**
## Description

This PR adds a PR template (see [diff](https://github.com/juztamau5/bubblewars-app/commit/pr-template)), demonstrated here. By adding a template, the author is guided through the process of making the contribution as reviewable and understandable as possible.

I've included the following fields:

### *Description*

Clearly important.

### *Motivation and Context*

Description is important, but I've found that having "Motivation and context" as the next field encourages the author to think about the contribution on a higher level. In otherwords, if description is the "what", motivation and context is the "why". Too often I see the "why" forsaken, so including it here forces thought into the importance of the PR and argue for why it should be accepted.

### *Related PRs*

"Related PRs" is also important as a category due to "PR linking". Cross-linking PRs, which GitHub does automatically, is an important part of the process in hunting down where a change came from and why it was made.

### *How Has This Been Tested?*

Finally, we challenge the author to actually test the change. It's usually assumed that PRs are tested, but this forces the author to be honest about their testing effort, and can be used to identify possible other test scenarios that should be considered.

## Motivation and Context

As collaboration on bubblewars ramps up, it's important to focus on the most important part of the code: paradoxically, the reviewers. Code review results not just in immaculate code quality, but improves the ability of all participants in the review process.

In other words, "We write code for people instead of computers, because people change the world, not computers".

And it's a cool flex to be able to walk into a new project and confidently bust out code quality like no one has seen before.

## Related PRs

First PR using the new template. Woo!

## How Has This Been Tested?

I introduced basically the same template when I started at Aclima, and it's been transformative in the quality of code across our entire firmware team.

Most importantly, it impacted a senior colleague, who started as a hesitant developer, and now I'm simply blown away by the quality of code they write. If I can bring the same improvement to others I'll consider my job a success.

### CODE REVIEW

**Comment by TheExpandingPolytope** [comment]
Amazing
